docs/subvolume.md

```diff
@@ -39,8 +39,18 @@ kubectl rook-ceph subvolume ls --stale
 ## delete
 
 ```bash
-kubectl rook-ceph subvolume delete ocs-storagecluster csi-vol-427774b4-340b-11ed-8d66-0242ac110004
+kubectl rook-ceph subvolume delete ocs-storagecluster-cephfilesystem csi-vol-427774b4-340b-11ed-8d66-0242ac110005
 
-# Info: subvolume "csi-vol-427774b4-340b-11ed-8d66-0242ac110004" deleted
+# Info: subvolume csi-vol-427774b4-340b-11ed-8d66-0242ac110004 deleted
+# Info: omap object:"csi.volume.427774b4-340b-11ed-8d66-0242ac110005" deleted
+# Info: omap key:"csi.volume.pvc-fca205e5-8788-4132-979c-e210c0133182" deleted
+
+```
+
+```bash
+kubectl rook-ceph subvolume delete ocs-storagecluster-cephfilesystem csi-vol-test
```

Comment:
`csi-vol-test` is not a proper format use `csi-vol-427774b4-340b-11ed-8d66-0242ac112345`